cmake/setup_write_config.cmake

```diff
@@ -56,6 +57,10 @@ MACRO(_summary)
   # Only write to summary.log:
   FILE(APPEND ${_log_summary} "${ARGN}")
 ENDMACRO()
+MACRO(_featurelog)
```

Comment:
that's the code style for other macros above. If you want, i can apply it to all within this file. I just wanted to be consistent with what is written already.